q2lint.py

```diff
@@ -35,11 +35,18 @@ def main():
                           pathlib.Path('.').glob('**/*/package.json'))
     if npm_packages:
         import subprocess
+        import os
 
         for package in npm_packages:
             pkg_path = package.parent
-            cmd = 'cd %s && npm i && npm run build -- --bail && (git diff ' \
-                  '--quiet */bundle.js || bash -c "exit 42")' % pkg_path
+            install = 'cd %s && npm i --silent' % pkg_path
+            cmd = subprocess.run(install, shell=True,
+                                 stdout=open(os.devnull, 'wb'))
+            if cmd.returncode != 0:
+                errors.append('npm install failed: %s' % cmd.stderr)
```

Comment:
Thanks, looks like a comment didn't actually get posted with the review --- can you drop the `continue`? That way it will pass through to the build step, which will give us detailed failure indicators.